graylog2-web-interface/src/components/event-definitions/replay-search/EventInfoBar.tsx

```diff
@@ -0,0 +1,218 @@
+import useAppSelector from 'stores/useAppSelector';
+import { selectHighlightingRules } from 'views/logic/slices/highlightSelectors';
+import { conditionToExprMapper } from 'hooks/useHighlightValuesForEventDefinition';
+import type { StaticColor } from 'views/logic/views/formatting/highlighting/HighlightingColor';
+
+const AlertTimestamp = styled(Timestamp)(({ theme }) => css`
+  color: ${theme.colors.variant.darker.warning}
+`);
+
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+  user-select: none;
+  gap: 5px;
+`;
+
+const Item = styled.div`
+  display: flex;
+  gap: 5px;
+  align-items: flex-end;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`;
+
+const Row = styled.div`
+  display: flex;
+  gap: 8px;
+  flex-wrap: wrap;
+`;
+const useHighlightingRules = () => useAppSelector(selectHighlightingRules);
+
+const EventInfoBar = () => {
+  const [open, setOpen] = useState<boolean>(true);
+  const allNotifications = useStore(EventNotificationsStore, ({ all }) => {
+    return all.reduce((res, cur) => {
+      res[cur.id] = cur;
+
+      return res;
+    }, {});
+  });
+  const { eventData, eventDefinition, aggregations, isEventDefinition } = useAlertAndEventDefinitionData();
+
+  const toggleOpen = (e) => {
+    e.stopPropagation();
+    setOpen((cur) => !cur);
+  };
+
+  // const executeEvery = extractDurationAndUnit(ev, TIME_UNITS);
+  const searchWithin = extractDurationAndUnit(eventDefinition.config.search_within_ms, TIME_UNITS);
+  const executeEvery = extractDurationAndUnit(eventDefinition.config.execute_every_ms, TIME_UNITS);
+
+  const notificationList = useMemo(() => {
+    return eventDefinition.notifications.reduce((res, cur) => {
+      if (allNotifications[cur.notification_id]) {
+        res.push((allNotifications[cur.notification_id]));
+      }
+
+      return res;
+    }, []);
+  }, [eventDefinition, allNotifications]);
+
+  const isEDUpdatedAfterEvent = !isEventDefinition && moment(eventDefinition.updated_at).diff(eventData.timestamp) > 0;
+  const highlightingRules = useHighlightingRules();
+
+  const highlightingColors = useMemo(() => {
+    const aggregationsSet = new Set(aggregations.map(({ fnSeries, value, expr }) => `${fnSeries}${expr}${value}`));
+
+    return highlightingRules.reduce((res, rule) => {
+      const { field, value, condition } = rule;
+      const color = rule.color as StaticColor;
+      const expr = conditionToExprMapper?.[condition];
+
+      if (expr) {
+        const key = `${field}${expr}${value}`;
+
+        if (aggregationsSet.has(key)) {
+          res[key] = color.color;
+        }
+      }
+
+      return res;
+    }, {});
+  }, [aggregations, highlightingRules]);
+
+  return (
+    <FlatContentRow>
+      <Header>
+        <Button bsStyle="link" className="btn-text" bsSize="xsmall" onClick={toggleOpen}>
+          <Icon name={`caret-${open ? 'down' : 'right'}`} />&nbsp;
+          {open ? 'Less details' : 'More details'}
```

Comment:
Instead of "Less details" and more details, we could name it "Show event definition details" / "Hide event definition details". It would make it a bit more descriptive.